attestation-agent/kbs_protocol/src/client/rcar_client.rs

```diff
@@ -334,6 +334,7 @@ mod test {
 
     const CONTENT: &[u8] = b"test content";
 
+    #[ignore]
```

Comment:
The test knows to ignore protocol version error so you could just bump https://github.com/confidential-containers/guest-components/blob/8e39f5ac58066910c60c698d4dd5ff60654d07d6/attestation-agent/kbs_protocol/src/client/mod.rs#L51 to get the same result.

Reply:
Hm good point. There's an interesting question here about whether we should bump the KBS protocol version in this PR. The first version of this PR relied on changes in the KBS protocol (to specify all the TEEs in the first part of the RCAR handshake), but this version does not. That said, multi-device guest-components won't be compatible with an older version of Trustee because the format of the evidence is different. I'm not sure exactly the best way to capture this.